hr_evaluation_360/README.rst

```diff
@@ -0,0 +1,79 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=================
+Hr Evaluation 360
+=================
+
+360-degree feedback
+
+Installation
+============
+
+To install this module, you need to:
+
+#. Do this ...
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. Go to ...
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to ...
```

Comment:
Go ahead and write something about the Configuration steps to perform, and guide a new user to the steps to use it.